tpm2/constants.go

```diff
@@ -393,6 +393,9 @@ const (
 	CmdGetCapability              tpmutil.Command = 0x0000017A
 	CmdGetRandom                  tpmutil.Command = 0x0000017B
 	CmdHash                       tpmutil.Command = 0x0000017D
+	CmdHashSequenceStart          tpmutil.Command = 0x00000186
```

Comment:
Nit: Could these be inserted in the enum at a position corresponding to increasing values?

Reply:
When I go look into a enum the value doesn't matters to me, the name does.
Usually enums are ordered/created with related flags/command_codes near to each other, but TCG didn't went that way.
For me as an user I would like to find hash/sequence values near each other:
```